app/javascript/styles_new/mastodon/accounts.scss

```diff
@@ -113,8 +113,8 @@
   }
 
   .current {
-    color: var(--color-bg-inverted);
-    background: var(--color-text-on-inverted);
```

Comment:
I could've just swapped the text and bg tokens here, as the actual issue was that these were applied to the wrong property.

But I thought rather than use the dedicated `inverted` tokens, it made more sense to literally swap the same text and background colours that are used for the regular links, as those might look a bit different.